common/src/main/resources/common/tenant.feature

```diff
@@ -4,6 +4,7 @@ Feature: Tenants
     * url baseUrl
     * configure retry = { count: 2, interval: 5000 }
     * configure readTimeout = 3000000
+    * call login admin
```

Comment:
why this change?

Reply:
![image](https://github.com/folio-org/folio-integration-tests/assets/101543142/ea1692f9-73d1-4862-8852-dfb27a84506e)
 i've got these errors after running a test without these changes

Reply:
I ran test against our perf rancher env